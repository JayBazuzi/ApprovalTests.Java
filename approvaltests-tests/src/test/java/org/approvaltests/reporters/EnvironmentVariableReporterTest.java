package org.approvaltests.reporters;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class EnvironmentVariableReporterTest {
    @Test
    public void testEnvironmentVariable() {
        var reporter = EnvironmentVariableReporter.INSTANCE.getReporter();

        Assertions.assertInstanceOf(MultiReporter.class, reporter);
        var multiReporter = (MultiReporter) reporter;
        var reporters = multiReporter.getReporters();

        Assertions.assertEquals(2, reporters.length);
        Assertions.assertInstanceOf(DiffReporter.class, reporters[0]);
        Assertions.assertInstanceOf(QuietReporter.class, reporters[1]);
    }
}
