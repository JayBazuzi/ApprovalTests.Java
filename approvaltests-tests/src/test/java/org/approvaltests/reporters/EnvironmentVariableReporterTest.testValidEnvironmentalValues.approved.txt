AraxisMergeReporter
AutoApproveReporter
AutoApproveWhenEmptyReporter
BeyondCompareReporter
ClipboardReporter
CodeCompareReporter
DelayedClipboardReporter
DiffMergeReporter
DiffReporter
FileCaptureReporter
ImageReporter
ImageWebReporter
IntelliJReporter
JunitReporter
KDiff3Reporter
KaleidoscopeDiffReporter
MeldMergeReporter
P4MergeReporter
PitReporter
QuietReporter
TestNgReporter
TextWebReporter
TkDiffReporter
TortoiseDiffReporter
VisualStudioCodeReporter
WinMergeReporter
WindowsDiffReporter
